docs/guide/execution-checkpoints.md

```diff
@@ -58,6 +58,8 @@ The keys of these maps are strings which should be unique per network inside of
 Editing them by hand is not encouraged (but if it's needed their names must not match any asset in the same network).
 Currently it's possible to edit these files by hand but it may be decided to disallow this in the future.
 
+Note: Checkpoints will not be created or modified when script is running in run mode.
```

Comment:
Same as above, i think we only create new checkpoints, or read from existing ones. Right?

Reply:
no it can be modified if deletion occurs